documentation/asciidoc/computers/raspberry-pi/boot-eeprom-rpi4.adoc

```diff
@@ -2,7 +2,7 @@
 
 Raspberry Pi 5, Raspberry Pi 4, 400, Compute Module 4, and Compute Module 4S computers use an EEPROM to boot the system. All other models of Raspberry Pi computer use the `bootcode.bin` file located in the boot filesystem.
 
-NOTE: The scripts and pre-compiled binaries used to create the `rpi-eeprom` package which is used to update the Raspberry Pi 4 bootloader and VLI USB controller EEPROMs is available https://github.com/raspberrypi/rpi-eeprom/[on GitHub].
+NOTE: The scripts and pre-compiled binaries used to create the `rpi-eeprom` package which is used to update the Raspberry Pi 4 bootloader and VLI USB controller EEPROMs are available https://github.com/raspberrypi/rpi-eeprom/[on GitHub].
```

Comment:
```suggestion
NOTE: You can find the scripts and pre-compiled binaries used to create the `rpi-eeprom`  in the https://github.com/raspberrypi/rpi-eeprom/[rpi-eeprom GitHub repository].
```
Suggestion: There's a lot of meat in this sandwich. We should probably make it vegetarian.